packages/certs-v5/lib/endpoints.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+const MULTIPLE_SNI_ENDPOINT_FLAG = 'allow-multiple-sni-endpoints'
```

Comment:
You have this feature name hardcoded in `add.js` above. It could make sense to create a features module to contain these constants and maybe even the logic to check if a feature is enabled.

Reply:
Good idea! just pushed this up